src/modules/rep.ts

```diff
@@ -158,6 +158,12 @@ export class RepModule extends Module {
 		description: 'See who has the most reputation',
 	})
 	async leaderboard(msg: Message) {
+		function getLeaderboardEmoji(index: number) {
```

Comment:
Could just have an array and `||` instead of a whole other function. It's cleaner.